src/storage-client/src/client.rs

```diff
@@ -483,9 +483,6 @@ impl RustType<proto_storage_response::ProtoStatusUpdate> for StatusUpdate {
 #[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
 pub enum StorageResponse<T = mz_repr::Timestamp> {
     /// A list of identifiers of traces, with new upper frontiers.
-    ///
-    /// TODO(teskje): Consider also reporting the previous upper frontier and using that
-    /// information to assert the correct implementation of our protocols at various places.
```

Comment:
I'm pretty sure nobody will have the time to attempt this. Also it probably produces more noise than it's worth.